fixup-mountpoints

```diff
@@ -14,6 +14,106 @@ shift
 echo "Fixing mount-points for device $DEVICE"
 
 case "$DEVICE" in
+    "ghost")
+        sed -i \
+            -e 's block/platform/msm_sdcc.1/by-name/modem mmcblk0p1 ' \
+            -e 's block/platform/msm_sdcc.1/by-name/persist mmcblk0p23 ' \
+            -e 's block/platform/msm_sdcc.1/by-name/system mmcblk0p38 ' \
+            -e 's block/platform/msm_sdcc.1/by-name/userdata mmcblk0p40 ' \
+            -e 's block/platform/msm_sdcc.1/by-name/fsg mmcblk0p25 ' \
+            -e 's block/platform/msm_sdcc.1/by-name/pds mmcblk0p28 ' \
```

Comment:
Any reason you haven't added cache, boot, and recovery entries here? They were in the old PR: https://github.com/mer-hybris/hybris-boot/pull/72/commits/2a699e174a5cb7a6cd8caac14b920c75eb63b7ba#diff-088581b2b95922dfc7a1f12ef8de3ba7R173